charts/opentelemetry-operator/values.yaml

```diff
@@ -16,10 +16,10 @@ imagePullSecrets: []
 manager:
   image:
     repository: ghcr.io/open-telemetry/opentelemetry-operator/opentelemetry-operator
-    tag: v0.56.0
+    tag: v0.58.0
```

Comment:
Should this be 0.59.0?

Reply:
nope https://github.com/open-telemetry/opentelemetry-operator/releases

Reply:
@Allex1  @TylerHelmuth  As v0.59.0 has been released for the operator, should we modify this PR to use the same ?

Reply:
@avadhut123pisal Done